docs/en/latest/how-to-build.md

```diff
@@ -54,6 +54,23 @@ $ sudo yum --showduplicates list apisix
 $ sudo yum install apisix
 ```
 
+### Installation via RPM Offline Package(CentOS 7)
+
+Download APISIX offline RPM package to `./apisix` folder
+
+```shell
+$ sudo mkdir -p apisix
+$ sudo yum install -y https://repos.apiseven.com/packages/centos/apache-apisix-repo-1.0-1.noarch.rpm
+$ sudo yum clean all && yum makecache
+$ sudo yum install -y --downloadonly --downloaddir=./apisix apisix
```

Comment:
```suggestion
# Create a folder `apisix` to store the RPM packages
sudo mkdir -p apisix
# Install apisix repo
sudo yum install -y https://repos.apiseven.com/packages/centos/apache-apisix-repo-1.0-1.noarch.rpm
# Rebuild YUM cache
sudo yum clean all && yum makecache
# Download apisix and its deps package to the `apisx` folder
sudo yum install -y --downloadonly --downloaddir=./apisix apisix
```
Shall we remove `$` ? @juzhiyuan 
IMO, `$` is used to distinguish between `command` and its `output` like
```bash
$ uptime 
17:24:16 up 3 days,  8:06,  1 user,  load average: 0.00, 0.01, 0.05
```

Reply:
`uptime` is a command, right? 😄

Reply:
yep